serde_v8/src/de.rs

```diff
@@ -268,8 +268,13 @@ impl<'de, 'a, 'b, 's, 'x> de::Deserializer<'de>
       Some(names) => from_v8(self.scope, names.into()).unwrap(),
       None => vec![],
     };
-    let keys: Vec<v8::Local<v8::Value>> =
-      keys.drain(..).map(|x| x.into()).collect();
+    let keys: Vec<v8::Local<v8::Value>> = keys
+      .drain(..)
+      .map(|x| x.into())
+      // Filter keys to drop keys whose value is undefined
+      // TODO: optimize, since this doubles our get calls
```

Comment:
How big is the difference?

I think the checking for undefined can also be done by dereferencing the `Local<Value>` once or twice and testing for equality. It's hacky of course, but good enough for a try out.

Reply:
It's easy to optimize but I'd rather not getting into micro-optimizing serde_v8 without restoring its benches.

So the TODO mainly serves as a motivation to restore `serde_v8`'s micro-benches.

> How big is the difference?

Don't know, it's not benched, probably a few nanoseconds per get.